Memeolist/Memeolist/view/list-meme/MemesListViewController.swift

```diff
@@ -94,7 +92,7 @@ class MemesListViewController: UITableViewController {
         if (segue.identifier == "CommentsSegue") {
             if let button = sender as? UIButton, let cell = button.superview?.superview as? MemesTableViewCell {
                 let destination = segue.destination as? CommentsViewController
-                destination?.meme = memes?.first(where: { $0.id == cell.memeId })
+                destination?.meme = memes.first(where: { ($0?.id)! == cell.memeId! }) as? MemeDetails
```

Comment:
@wtrocki I don't know how to make it less ugly :(

Reply:
We can assign id as tag to the cell and fetch it.